packages/slice-machine/components/AppLayout/Navigation/index.tsx

```diff
@@ -1,4 +1,4 @@
-import useWindowSize from "../../../hooks/useWindowSize";
+import useWindowSize from "src/hooks/useWindowSize";
```

Comment:
should still be relative import, i think absolute import doesn't work sometimes.

Reply:
Some absolute imports are not working because of alias resolution is not the same for the front and back